starkinfra/dynamicbrcode/__dynamicbrcode.py

```diff
@@ -0,0 +1,250 @@
+from json import dumps
+from ..utils import rest
+from ..utils import parse
+from starkcore.utils.api import api_json
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date
+
+
+class DynamicBrcode(Resource):
+    """# DynamicBrcode object
+    BR codes store information represented by Pix QR Codes, which are used to
+    send or receive Pix transactions in a convenient way.
+    DynamicBrcodes represent charges whose information can change at any time
+    since all information needed for the payment is requested dynamically.
```

Comment:
DynamicBrcodes represent charges with information that can change at any time,
since all data needed for the payment is requested dynamically to an URL stored
in the BR Code. Stark Infra will receive the GET request and forward it to your registered endpoint with a GET request containing the UUID of the BR code for identification.